xbmc/video/videosync/VideoSyncOsx.cpp

```diff
@@ -73,18 +81,27 @@ float CVideoSyncOsx::GetFps()
   return m_fps;
 }
 
+void CVideoSyncOsx::OnLostDevice()
+{
+  if (!m_displayLost)
+  {
+    m_displayLost = true;
+    m_lostEvent.Wait();
+  }
+}
+
 void CVideoSyncOsx::OnResetDevice()
 {
-  m_abort = true;
+  m_displayReset = true;
 }
 
-void CVideoSyncOsx::VblankHandler(int64_t nowtime)
+void CVideoSyncOsx::VblankHandler(int64_t nowtime, int32_t timebase)
```

Comment:
@FernetMenta should i call updateclock with CurrentHostCounter instead of nowtime?